CSE115A/rideshare-project
# Made landing page to match figma design

Problem
=======
As a user, I want to visually see which rideshare app is cheaper so I can speed up the checkout process

[Link to Ticket](https://trello.com/c/FTiW36a0/66-5-pts-create-landing-page)

Solution
========
@sarahAdamson99 and I created and designed the text boxes and headings to match the figma design

Change summary:
---------------
* View previous commits made by Sarah, Isai, and me

Steps to Verify:
----------------
1. Checkout the branch
2. Install dependencies and start local server
3. Verify that the front-end is there
4. Verify that

Screenshots (optional):
-----------------------
Show-n-tell images/animations here
![spc](https://user-images.githubusercontent.com/37026447/98880434-961c3200-243c-11eb-8950-fef5e744b57a.PNG)
